src/GraphQL.Tests/Bugs/BugWithCustomScalarsInDirective.cs

```diff
@@ -24,6 +24,7 @@ public class BugWithCustomScalarsInDirectiveSchema : Schema
     public BugWithCustomScalarsInDirectiveSchema(IServiceProvider provider, BugWithCustomScalarsInDirectiveQuery query)
         : base(provider)
     {
+        this.RegisterBCLScalars();
```

Comment:
Not all tests require this. For first run ~350 tests failed for 5 platforms so there are ~70 "real" tests that use custom scalars.